src/components/EditCoursePage/EditCourseForm.jsx

```diff
@@ -69,7 +69,7 @@ export class BaseEditCourseForm extends React.Component {
     const stoppingRunReview = prevProps.courseSubmitInfo.isSubmittingRunReview &&
                               !courseSubmitInfo.isSubmittingRunReview;
     const hasCourseErrors = courseSubmitInfo.errors &&
-                            courseSubmitInfo.errors !== {} &&
+                            Object.keys(courseSubmitInfo.errors).length &&
```

Comment:
Unrelated change - you can't compare objects like they are dicts, because dicts aren't a real thing in JS.